x/ccv/consumer/keeper/keeper.go

```diff
@@ -428,8 +429,36 @@ func (k Keeper) DeletePendingDataPackets(ctx sdk.Context) {
 }
 
 // AppendPendingDataPacket appends the given data packet to the pending data packets in store
-func (k Keeper) AppendPendingPacket(ctx sdk.Context, packet types.ConsumerPacket) {
+func (k Keeper) AppendPendingPacket(ctx sdk.Context, packet ...types.ConsumerPacket) {
 	pending := k.GetPendingPackets(ctx)
-	list := append(pending.GetList(), packet)
+	list := append(pending.GetList(), packet...)
 	k.SetPendingPackets(ctx, types.ConsumerPackets{List: list})
 }
+
+// GetHeightToValsetUpdateIDs returns all height to valset update id mappings in store
+func (k Keeper) GetHeightToValsetUpdateIDs(ctx sdk.Context) []types.HeightToValsetUpdateID {
+	heightToVCIDs := []types.HeightToValsetUpdateID{}
+	k.IterateHeightToValsetUpdateID(ctx, func(height, vscID uint64) bool {
+		hv := types.HeightToValsetUpdateID{
+			Height:         height,
+			ValsetUpdateId: vscID,
+		}
+		heightToVCIDs = append(heightToVCIDs, hv)
+		return true
```

Comment:
This will need to be updated after https://github.com/cosmos/interchain-security/pull/459 is merged.